sidebars.ts

```diff
@@ -310,6 +310,7 @@ module.exports = {
                 'send-data/hosted-collectors/http-source/traces',
                 'send-data/hosted-collectors/http-source/generate-new-url',
                 'send-data/hosted-collectors/http-source/troubleshooting',
+                'send-data/hosted-collectors/http-source/zoom',
```

Comment:
I'd move this under `send-data/hosted-collectors/http-source/traces` because `generate-new-url` and `troubleshooting` are not sources.

Reply:
Created a different folder named "Webhooks Sources" and Zoom source is kept under this. So that we be inline with  Amazonn, Microsoft , Google, and Cloud Sylogs which are an individual folders.